### PR TITLE
Vue transition

### PR DESCRIPTION
**Pull request by Marc-Lorenz**

### CODE REVIEW

**Review by FHellmann — CHANGES_REQUESTED** [review]
- [x] after opening image(s) they are placed at the top left corner and not scaled to the screen size and placed in the center
- [x] pan does not work
- [x] graph points are not moveable
- [x] points do not show hovering effect
- [x] loading existing annotation does not work
- [x] in the left menu: switch model and view to have same outline as in original version

File: src/graph/graph.ts [hunk 1]
```diff
@@ -23,23 +27,34 @@ export class Graph<P extends Point2D> {
...
    * @param {() => P} newObject - A function to create a new point object.
    * @returns {Graph<P>} - A new Graph instance.
    */
-  static fromJson<P extends Point2D>(
-    jsonObject: P[],
-    newObject: (id) => P,
-  ): Graph<P> {
+  static fromJson<P extends Point2D>(jsonObject: P[], newObject: (id: number) => P): Graph<P> {
     return new Graph<P>(
       jsonObject.map((dict) => {
```
Comment (FHellmann) [inline src/graph/graph.ts:51]: I receive the following error message when trying to open annotations.

```
Uncaught TypeError: jsonObject.map is not a function
    at Graph.fromJson (graph.ts:52:18)
    at LoadSaveActions.vue:65:47
    at Array.forEach (<anonymous>)
    at LoadSaveActions.vue:64:32
    at Array.forEach (<anonymous>)
    at reader.onload (LoadSaveActions.vue:54:31)
```

**Review by FHellmann — CHANGES_REQUESTED** [review]
- [x] Error when loading annotation
- [x] Shortcuts not working (only STRG+O and SHIFT+SCROLL is working)

File: src/graph/graph.ts (hunk 1 above)
Comment (Marc-Lorenz) [reply src/graph/graph.ts:51]: We changed the format from

[Point]

to [[Point]]

should we support both?

Comment (FHellmann) [reply src/graph/graph.ts:51]: It would be good to be backward compatible. At least for now. In the future we can remove it.

**Review by FHellmann — CHANGES_REQUESTED** [review]
- [x] Delete Left Eye: It deletes the right eye iris point instead of the left eye iris point.
- [x] Delete Right Eye: It doesn't delete the right eye iris point.
- [x] In the menu under category "File" the "save" button sometimes appears.

**Review by FHellmann — APPROVED** [review]

**Pull request merged**